pennylane/qnn/keras.py

```diff
@@ -51,6 +54,9 @@ class KerasLayer(Layer):
     `Model <https://www.tensorflow.org/api_docs/python/tf/keras/Model>`__ classes for
     creating quantum and hybrid models.
 
+    .. warning::
+        This class is deprecated because Keras 2 is no longer actively maintained.  Please consider using a different machine learning framework instead of TensorFlow/Keras 2.
```

Comment:
```suggestion
        This class is deprecated because Keras 2 is no longer actively maintained.  Please consider using torch instead of TensorFlow/Keras 2.
```